src/main/charts/jira/templates/config-jvm.yaml

```
@@ -10,6 +10,11 @@ data:
     {{- range .Values.jira.additionalJvmArgs }}
     {{ . }}
     {{- end }}
+    {{- if hasSuffix "m" .Values.jira.resources.container.requests.cpu }}
+    -XX:ActiveProcessorCount={{ div (trimSuffix "m" .Values.jira.resources.container.requests.cpu) 1000 | default 1 }}
+    {{ else }}
+    -XX:ActiveProcessorCount={{ .Values.jira.resources.container.requests.cpu }}
+    {{ end }}
```

```suggestion
    {{- end }}
```
to remove the empty line